gcs-action/src/gcs.ts

```diff
@@ -0,0 +1,202 @@
+import {
```

Comment:
Needs license header. For example: https://github.com/GoogleCloudPlatform/github-actions/blob/2a9cbc618184da4718e0d16a1b0a055a4f90f66e/setup-gcloud/src/version-util.ts#L1

Reply:
Done